cmd/apiserver-boot/boot/glide.go

```diff
@@ -40,7 +40,7 @@ func AddGlideInstallCmd(cmd *cobra.Command) {
 }
 
 func fetchGlide() {
-	c := exec.Command("glide", "install", "--strip-vendor")
+	c := exec.Command("glide", "install", "--strip-vendor", "--strip-vcs")
```

Comment:
`strip-vcs` is the default and deprecated btw - might make sense to not add it in as it will need removing some time in future.